UsageCollectorWorkerService/Services/LowLevelCollecting/LowLevelCollectingService.cs

```diff
@@ -23,10 +23,10 @@ public async Task<byte> GetCpuUsageInPercent()
         string[] lines = output.Split("\n");
         string[] memory = lines[3].Split(" ", StringSplitOptions.RemoveEmptyEntries);
         
-        return (byte)(100 - Convert.ToByte(memory[11].Split(",", StringSplitOptions.RemoveEmptyEntries)[0]));
+        return 100 - Convert.ToInt32(memory[11].Split(",", StringSplitOptions.RemoveEmptyEntries)[0]);
```

Comment:
It would be more readable to add this value to some variable with descriptive name. Or / plus add comment what this formula means.